cypress/e2e/homepage.spec.ts

```diff
@@ -0,0 +1,217 @@
+import { ecosystemLinks, resourcesLinks } from "../fixture/homepage.config";
+
+const viewports = ["iphone-xr", "macbook-16"];
+
+viewports.forEach((viewport) => {
+  context(`/Homepage on ${viewport}`, () => {
```

Comment:
`/Homepage` might be misleading, since this url route doesnt exist
```suggestion
  context(`Homepage on ${viewport}`, () => {
```